Sources/Fuzzilli/Core/MultiEngine.swift

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+/// Wraps multiple engines into one, which can be initialized given a
+/// WeightedList. This can then switch engines to use, or use a more
+/// complicated heuristic.
+public class MultiEngine: ComponentBase, FuzzEngine {
```

Comment:
Cool! So I think I'd like this to have an `activeEngine` member which is then used for a certain (configurable?) number of `fuzzOne` rounds before it is replaced by `engines.randomElement()`. Moreover, the initial `active` engine should be configurable. This way, you can e.g. say that you want to start with N rounds of fuzzing from the HybridEngine/GeneratorEngine to generate some Corpus before running the MutationEngine. WDYT?

Reply:
Yeah, I had something similar for testing and actually forgot to remove the `iterationCount` variable :D